### ISSUE TITLE
Color Themes

### ISSUE THREAD

**Issue by EPonyA** [IlanCosman/tide#116]
My overall favorite prompt currently is probably [Bobthefish](https://github.com/oh-my-fish/theme-bobthefish), but I've been playing with Tide and I like your project much more in some ways.

One issue I've had is that neither the true color nor the 16 color settings happen to look good with my Gruvbox terminal. Using 16, the `brwhite` color is pretty much unreadable for text in my theme. `white` is slightly better, but I had the best experience changing it to `black`.
Default on Rainbow + 16 Color
![image](https://user-images.githubusercontent.com/1811218/110254103-b6c44680-7f5b-11eb-9ef8-52089d399b52.png)
With my changes in `fish_variables`:
![image](https://user-images.githubusercontent.com/1811218/110254174-09056780-7f5c-11eb-97e2-bee153b79737.png)
![image](https://user-images.githubusercontent.com/1811218/110254207-35b97f00-7f5c-11eb-8362-d413ff695927.png)
Default on Rainbow + True Color
![image](https://user-images.githubusercontent.com/1811218/110255219-98147e80-7f60-11eb-8184-de802dc31f18.png)

Bobthefish provides a function to change the color scheme to quite a few popular defaults for its true-color setting (Dracula, Gruvbox, Nord, etc.), which allows me to instantly set up a prompt I find pretty readable.
![image](https://user-images.githubusercontent.com/1811218/110254668-0dcb1b00-7f5e-11eb-807c-583c9628527c.png)

I'd love something similar in Tide!

**Comment by IlanCosman** [comment]
Thanks for opening this feature request 😄 Unfortunately I don't think I'll be implementing anything like this.
- Truecolor mode always makes the prompt look good, but might not fit in with the rest of your terminal.
- 16color mode always makes the prompt fit in with the terminal, but might not look good if, well, certain colors from your color scheme just don't look good. If your color scheme made the mistake of having a color be unreadable, that's on them.

Simply put, I don't feel like making color configs for every theme out there--that's the point of 16color mode. And I don't think I can have the community, like you, do it, because I can't trust that you'll stick around and update it for new items and changes.

Truecolor and 16color cover probably 2/3 of themes and 90% of users. Basically you're on your own here, sorry 😞
